src/renderer/main.js

```diff
@@ -56,12 +56,12 @@ Vue.use(VueHighCharts);
 Vue.use(VueI18n);
 require('vue-flash-message/dist/vue-flash-message.min.css');
 
-// Vue.use(VueNativeSock, 'ws://localhost:62433/ws', {
-Vue.use(VueNativeSock, 'wss://testnet.aphelion-neo.com:62443/ws', {
+Vue.use(VueNativeSock, network.getSelectedNetwork().websocketUri, {
```

Comment:
Is this needed at all anymore? Won’t authenticated wrapper do this before mount?